docs/development/contributing_guide.md

```diff
@@ -48,16 +48,16 @@ can find many good git tutorials on the web.
 
 # 4. Install the dependencies
 
-Once you have installed Python 3 and added the source, please open a terminal and
-setup a *virtualenv*, as follows:
+Synapse uses the [poetry](https://python-poetry.org/) project to lock its dependencies
```

Comment:
```suggestion
Synapse uses the [poetry](https://python-poetry.org/) project to manage its dependencies
```

This sounds more natural to me, especially since it's supposed to also apply to "development environment".